release.sh

```diff
@@ -1,25 +1,39 @@
 #!/usr/bin/env bash
-set -eux
+set -euo pipefail
 
-if [ -z "$1" ]
+if [ -z "${1:-}" ]
```

Comment:
This `${1:-}` is semantically equal to `${1}`. Why do you propose to add `:-`?

Reply:
I typically use a construct like

```
if [ $# -lt 1 ]
```

Reply:
Hmm, that will allow empty strings `""` as an argument. How about something like https://github.com/zalando/intellij-swagger/pull/353/commits/45ff208327f68d9c204a6977eaa704c4772b23aa

Reply:
Ah yes. That's fine. I also tend to do these checks then also later to verify that no empty arguments are provided. So all is fine.